arch/arm64/src/common/arm64_copystate.c

```diff
@@ -102,14 +102,15 @@ int arm64_syscall_save_context(uint64_t * regs)
 
 #ifdef CONFIG_ARCH_FPU
```

Comment:
This whole module seems to be dead code, can it be removed ?

Reply:
yes, i think so.